Project.toml

```diff
@@ -28,4 +28,4 @@ RandomExtensions = "0.4.2"
 Singular_jll = "~403.206.600"
```

Comment:
should also refer to the latest Singular_jll: Singular_jll = "~403.207.700"

Reply:
Thank you. Then we first need https://github.com/JuliaPackaging/Yggdrasil/pull/7262 -- my bad